File: Java/src/Iniciante/Solucao_1006.java
import java.io.IOException;
import java.util.Scanner;

public class Solucao_1006{

    public static void main(String[] args) throws IOException {
        var input = new Scanner(System.in); //objeto Scanner para os inputs

        double pesoA = 2; //peso A, B e C
        double pesoB = 3;
        double pesoC = 5;
        double x, y, z, resultado;//declaração das variaveis

        x = input.nextDouble();//leitura dos inputs
        y = input.nextDouble();
        z = input.nextDouble();

        resultado = ((x * pesoA) + (y * pesoB) + (z * pesoC))/(pesoA + pesoB + pesoC );//resultado recebe a media
        System.out.printf("MEDIA = %.1f%n",resultado);//print com uma casa decimal
        input.close();//fechamento do input
    }

}

File: Python/Iniciante/Solucao_1006.py
try:
    def calcular_media(peso_a,peso_b,peso_c,x,y,z):# formula para o calculo da media
        return ((x*peso_a) + (y*peso_b) + (z*peso_c)) / (peso_a + peso_b + peso_c)

    peso_a = 2# declaração de peso A, B e C
    peso_b = 3
    peso_c = 5
    x = float(input())#leitura de X,Y e Z
    y = float(input())
    z = float(input())
    resultado = calcular_media(peso_a,peso_b,peso_c,x,y,z)# variavel resultado receber a função calcular media passando os argumentos acima
    print(f"MEDIA = {resultado:.1f}")# print com uma casa decimal
except (ValueError, TypeError):
    exit()